Automattic-Tracks-iOS/Event Logging/EventLoggingUploadManager.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import CocoaLumberjack
+import Sodium
+
+class EventLoggingUploadManager {
+
+    private struct Constants {
```

Comment:
I've been looking into this and it looks like this creating `struct` method to hold onto constants is more common for sharing things between files. So, I was curious why you opted for this in this case? I thought maybe it was to keep relevant constants together but then `Constants` name would not fit.

Reply:
Yep, it was just to keep the constants for this class together – I could've just exposed them as `static let ...` inside the class, but this just cut down the noise and made them pretty obvious when reading – basically it's just a way to namespace them out of the way.

Reply:
There's also a 2-year-old proposal by Diego to use an `enum` instead of a `struct` for grouping. https://github.com/wordpress-mobile/swift-style-guide/issues/12

You're welcome for making you more confused. 😄